app/views/messages/_messages_sidebar.html.erb

```diff
@@ -0,0 +1,33 @@
+<ul class="messages-sidebar" ng-class="{'focused': selectedMessage !== null}">
```

Comment:
Sidebar list of messages, complete with filtering and creating a new message links.